lua/leetcode-ui/question.lua

```diff
@@ -91,7 +91,16 @@ function Question:handle_mount()
     vim.cmd("$tabe " .. self.file:absolute())
 
     -- https://github.com/kawre/leetcode.nvim/issues/14
-    if self.lang == "rust" then pcall(require("rust-tools.standalone").start_standalone_client) end
+    if self.lang == "rust" then
+        local ok, tool = pcall(require, "rust-tools.standalone")
+        if not ok then
+            if not pcall(require, "rustaceanvim") then
+                log.warn("Plugin rust-tool.nvim either rustaceanvim for rust not found.")
+            end
+        else
+            pcall(tool.start_standalone_client)
+        end
+    end
```

Comment:
seems like  `rustaceanvim` doesn't have a standalone module.
so this could be simplified to just this

```lua
if self.lang == "rust" then
    pcall(function() require("rust-tools.standalone").start_standalone_client() end)
end
```

also you wouldn't want to log that the tool is not found when user don't use either of these tools

Reply:
Yeah, this is more concise.